src/box/lua/xlog.c

```diff
@@ -272,7 +272,7 @@ lbox_xlog_parser_open_pairs(struct lua_State *L)
 	    strncmp(cur->meta.filetype, "RUN", 3) != 0 &&
 	    strncmp(cur->meta.filetype, "INDEX", 5) != 0 &&
 	    strncmp(cur->meta.filetype, "DATA", 4) != 0 &&
```

Comment:
We had `.vyrun`/`.vyindex` vs `.run`/`.index` discussion some time ago and decided to go with `.run`/`.index` and use the same meta.filetype as the file extension.